TidepoolKit Example/Support/Extensions/Sample.swift

```diff
@@ -64,6 +110,21 @@ struct Sample {
                                         energy: TFoodDatum.Nutrition.Energy(350, .kilocalories))
         }
 
+        fileprivate static var cgmSettingsAlerts: TCGMSettingsDatum.Alerts {
+            let snooze = TCGMSettingsDatum.Alerts.Snooze(30, .minutes)
+            return TCGMSettingsDatum.Alerts(enabled: true,
+                                            urgentLow: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 55, units: .milligramsPerDeciliter, snooze: snooze),
+                                            urgentLowPredicted: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 55, units: .milligramsPerDeciliter, snooze: snooze),
+                                            low: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 70, units: .milligramsPerDeciliter, snooze: snooze),
+                                            lowPredicted: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 65, units: .milligramsPerDeciliter, snooze: snooze),
+                                            high: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 180, units: .milligramsPerDeciliter, snooze: snooze),
+                                            highPredicted: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 200, units: .milligramsPerDeciliter, snooze: snooze),
+                                            fall: TCGMSettingsDatum.Alerts.RateAlert(enabled: true, rate: 3, units: .milligramsPerDeciliterPerMinute, snooze: snooze),
+                                            rise: TCGMSettingsDatum.Alerts.RateAlert(enabled: true, rate: 3, units: .milligramsPerDeciliterPerMinute, snooze: snooze),
+                                            noData: TCGMSettingsDatum.Alerts.DurationAlert(enabled: true, duration: 30, units: .minutes, snooze: snooze),
```

Comment:
hm... curious about this alert as well. `noData` seems to implies that the communication is broken. For the Dexcom implementation we have `noReadings` which implies that communication exists but the sensor cannot provide a reading for the current state. Does `noData` map to `noReadings`? If yes, consider renaming.

Reply:
The term `noData` comes from the Dexcom API itself. Specifically, according to Dexcom documentation:

> "noData" – triggers when the display device stops receiving EGV data from the transmitter for a certain amount of time; corresponds to the No Readings alert

Reply:
That's my point. The alert is the `noReadings` alert. Again I see potential for confusion here by calling it `noData`.

Reply:
Is there a reason to keep it `noData`?

Reply:
Again, a published and public API, so changing is unlikely. Besides, there are a whole huge number of "naming differences" between Loop and the Tidepool data model. This is just something to understand when the TidepoolService, the "translation layer" from Loop to Tidepooo data model, is reviewed.